src/main/java/edu/pdx/cs/sftp/Client.java

```diff
@@ -579,40 +581,33 @@ void displayLogHistory() {
   }
 
   /**
-   * Deletes a file from the remote server. Can take one or multiple files in the format
-   * "testfile.txt, testfile2.txt"
+   * Deletes the specified file(s) from the remote server. Multiple file names can be included through a comma-separated
+   * list.
    *
-   * @param files -- The string read in main containing the names of the files.
+   * @param filename The string containing the name(s) of the file(s) to be deleted.
    */
-  void deleteRemoteFile(String files) {
-    String pwd;
-    if (files.contains(",")) {
-      // multiple files are wanted.
-      // take the string and separate out the files.
-      String removeWhitespace = files.replaceAll("\\s", "");
-      String[] arr = removeWhitespace.split(",");
-      String output = "";
+  void deleteRemoteFile(String filename) {
```

Comment:
Should we include an access modifier?

Reply:
I was wrestling with that! I was wondering if there were any benefits to giving it public vs. default access. The default modifier is accessible only within package, which provides more accessibility than private but is more restrictive than protected, and public. I've always been told to keep accessibility restrictions as strict as possible but curious to hear your thoughts!

Reply:
I don't know! 🙈 I had a similar mental battle on this. I like being explicit (particularly for Java), but I 100% agree it should be as restrictive as possible. I think leaving them as default is probably best.